README.md

```diff
@@ -36,6 +36,17 @@ Make sure this repository is in your `load-path`, and then:
 
 ## Configuration
 
+### Automatically turn on completions
+
+Completions will not be loaded automatically in `Swift` mode. To enable them, add the following to your emacs init file *(use `SPC-f-e-i` to open it in spacemacs)*:
```

Comment:
Can we change `Swift mode` to `swift-mode`? :)
Also not sure if the spacemacs comment is necessary - is it common for a spacemacs user to not know where the init file is?